# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxHarfBuzzShaper.h
# Commit: 768df8b3c770
# Full Hash: 768df8b3c7707d579965f1a37b073adbaa6ed7a4
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-08-16 16:01:29
# Regressor Bug: 1740530
# File Overlap Count: 1
# Description:
#   Bug 1740530 - patch 5 - Implement support for COLRv1 glyphs represented as acyclic graphs of paint records. r=gfx-reviewers,lsalzman
#   
#   Depends on D153870
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152041
# ==============================================================================

diff -r 4db61d87c172 -r 768df8b3c770 gfx/thebes/gfxHarfBuzzShaper.h
--- a/gfx/thebes/gfxHarfBuzzShaper.h	Tue Aug 16 10:39:51 2022 +0000
+++ b/gfx/thebes/gfxHarfBuzzShaper.h	Tue Aug 16 10:39:52 2022 +0000
@@ -100,6 +100,9 @@
                                  hb_font_funcs_t* aFontFuncs = nullptr,
                                  FontCallbackData* aCallbackData = nullptr);
 
+  hb_font_t* GetHBFont() const { return mHBFont; }
+  hb_face_t* GetHBFace() const { return hb_font_get_face(mHBFont); }
+
  protected:
   nsresult SetGlyphsFromRun(gfxShapedText* aShapedText, uint32_t aOffset,
                             uint32_t aLength, const char16_t* aText,